examples/user.py

```diff
@@ -36,14 +37,46 @@
     'password': 'myPass1234',
     'permissions': [
         {
-            'roleName': 'Read only',
-            'scopeUri': '/rest/scopes/6a6bb53c-5502-4f89-8573-cd1fb5b02a54',
+            'roleName': 'Infrastructure administrator',
+            'scopeUri': '/rest/scopes/6cf6d4da-1b5e-4322-9dff-6ef545ad700f'
```

Comment:
Automation script will not work with the hard coded uri. So use Scope class and get the available uri from OV.